doc/11-Concepts.md

```diff
@@ -0,0 +1,77 @@
+# Views
+
+## List Items
+The lists are one of the most prominent views for the monitoring. In the default view they follow this style:
+![DefaultAnatomy](res/ListItemAnatomy.jpg "A list item stripped down to the skeleton")
+
+### Visual
+Each element has a so-called visual. It's purpose is to highlight certain elements in long lists and give an intuitive overview about the underlying objects' condition. 
+In the case of host or service lists, for example, the state is displayed. This makes it immediately clear if and where there are problems that need to be taken care of, immediately from the overview.
```

Comment:
Minor nitpick: Remove `for example`.

`immediately` is repeated in the sentence, find a better wording.